package/CHANGELOG

```diff
@@ -21,6 +21,7 @@ The rules for this file:
   * 0.16.0
 
 Enhancements
```

Comment:
16.0 has been released, so this needs to be moved into the 16.1 section as a Fix